content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,250 @@
+:abbr:`BoM (Bill of Material)` and the new version. This ensures changes and tests are encapsulated
+in the revised :abbr:`BoM (Bill of Material)` and do not affect :abbr:`BoM (Bill of Material)` used
+in production. That is, until the :ref:`changes are applied <plm/eco/apply-changes>`.
+
+.. example::
+   View the summary of the differences between the current and revised keyboard :abbr:`BoMs (Bill of
+   Materials)` in the :guilabel:`BoM Changes` tab of the :abbr:`ECO (engineering change order)`.
+
+   .. image:: engineering_change_orders/BoM-changes.png
+      :align: center
+      :alt: View summary of component changes in the *BoM Changes* tab.
+
+Change operations
+=================
+
+To modify the work orders in a :abbr:`BoM (Bill of Material)`, click the :guilabel:`Revision` smart
+button on an active :abbr:`ECO (engineering change order)` to access the archived, new version of
+the :abbr:`BoM (Bill of Material)`. In the new :abbr:`BoM (Bill of Material)` version, switch to the
+:guilabel:`Operation Changes` tab and make changes to existing operations by selecting each
+operation line item and opening the corresponding :guilabel:`Open: Operations` pop-up window. Make
+changes to any of the fields in the pop-up and click :guilabel:`Save` once completed. Create new
+operations by clicking the :guilabel:`Add a line` button, and remove new operations by clicking the
+:guilabel:`Archive` button.
+
+.. note::
+   The :guilabel:`Operations` tab is not available by default. To enable it, navigate to
+   :menuselection:`Manufacturing --> Configuration --> Settings` and check the :guilabel:`Work
+   Orders` box.
+
+Once the changes are complete, navigate back to the :abbr:`ECO (engineering change order)` itself by
+clicking `ECO00X` in the breadcrumbs in the top left. On the :abbr:`ECO (engineering change order)`
+form, a new :guilabel:`Operation Changes` tab displays the differences between the current
+production :abbr:`BoM (Bill of Material)` and the new version. Modifications to the :abbr:`BoM (Bill
+of Material)` in an :abbr:`ECO (engineering change order)` will not affect the :abbr:`BoM (Bill of
+Material)` used in production. That is, until the :ref:`changes are applied
+<plm/eco/apply-changes>`.
+
+.. _plm/eco/apply-changes:
+
+Apply changes
+=============
+
+After verifying the changes, move the :abbr:`ECO (engineering change order)` to a :ref:`verification
+stage <plm/eco/stage-config>`. Once the approvers accept the changes, the :guilabel:`Apply Changes`
+button will become available. Click this button, and the :abbr:`ECO (engineering change order)`
+will be automatically closed.
+
+To ensure the changes are live, navigate to the :guilabel:`Revision` smart button. The :abbr:`BoM
+(Bill of Material)` will have the large red :guilabel:`Archived` tag removed. Additionally, navigate
+to the product form, switch to the :guilabel:`Miscellaneous` tab, and the :guilabel:`Version` field
+will be incremented.
```

Comment:
It might be good to give a more detailed explanation than just saying "incremented".